### DISCUSSION TITLE
How Yocto image generate syslog and dmesg files in folder /var/log

### DISCUSSION THREAD

**Discussion by fengyingchu0827** [OE4T/tegra-demo-distro#279] [General]
Hi,
May I ask how to have syslog and dmesg files in the folder of /var/log?
What I have to add in the bb files to build the image which have /var/log/syslog and /var/log/dmesg?

Thanks a lot.

**Comment by madisongh** [comment]
How to do this depends on how you configure your builds, which logging implementation you choose to use, and to a certain degree, which init manager you choose.  There are some hints about this in the [Yocto Project documentation](https://docs.yoctoproject.org/) - go there and choose the version/branch you're using, then click on the "All-in-one Mega Manual" link and search for "syslog" and "/var/log" on that page.

**Reply by fengyingchu0827** [reply]
Thank you. I can have syslog and kern.log in /var/log/. 
In demo-image-common.inc, set CORE_IMAGE_BASE_INSTALL += "rsyslog".
I also have VOLATILE_LOG_DIR = "no" in the .inc file.

Thanks a lot.
